frame/collective/src/migrations/v4.rs

```diff
@@ -136,7 +136,7 @@ pub fn post_migrate<P: GetStorageVersion + PalletInfoAccess, N: AsRef<str>>(old_
 		new_pallet_prefix.to_vec(),
 		|_| Ok(()),
 	);
-	assert!(new_pallet_prefix_iter.count() > 1);
+	assert!(new_pallet_prefix_iter.count() >= 1);
```

Comment:
the storage_version_key should be in the new_pallet_prefix, if we do `>=` then we only check that, not that some key has been moved

Reply:
in the pre-migrate we check there is at least one key starting with the old prefix, (or that the whole trie after the old prefix is empty).
So if there are some keys after the old prefix then there should be at least one key that have be moved.
```suggestion
	assert!(new_pallet_prefix_iter.count() > 1);
```

Reply:
> the storage_version_key should be in the new_pallet_prefix

yes, so we sure at least one key is there

> if we do `>=` then we only check that, not that some key has been moved

if the pallet is in a state where nothing is stored(except the storage_version_key), then using `>` will cause post_migrate to panic, I think

Reply:
ok then in the pre-migrate we shouldn't assert that there is something at the old prefix considering that this would panic in case the pallet is in a state where nothing is stored.

Reply:
The assertion has been removed from the `pre-migrate`

Reply:
ok good then